status_and_problems.tex

```diff
@@ -0,0 +1,132 @@
+\section{Status and Problems} \label{sec:statusAndProblems}
+
+\subsection{Status of the implementation} \label{sec:status}
+
+At the time this technote is in writing, March 2019, only \textbf{lsst\_distrib} package is released and distributed.
+
+
+\subsubsection{Packaging System} \label{sec:statusPkgs}
+
+The following technologies are already used in DM for packaging:
+
+\begin{itemize}
+\item {\bf Eups}: for the majority of DM packages. This included third party libraries that require customization, or that are not available in the conda channels. The lsst DM binary Eups repository is hosted at \url{https://eups.lsst.codes/}. 
+\item {\bf Sonatype Nexus}: for java based DM software developed packages
+\item {\bf Conda}: for packages public available, that can be used without any customization
+\item {\bf Pip/PiPy}: for python packages public available. In some cases for packages pip/PiPy is used for DM developed python packages, for example \textit{docsteady}.
+\end{itemize}
+
+
+\subsubsection{Distribution System} \label{sec:statusDistrib}
+
+The main tool used for distribution to operations and science community is \textbf{Docker}. 
+
```

Comment:
I think this is only true if we regard deployment on the LSP as “distribution to the science community”.

I suspect the vast majority of science users who download and run code on their own systems are not using Docker.

Reply:
Understood, I rephrase it.